src/main/scala/com/campudus/tableaux/database/domain/table.scala

```diff
@@ -3,14 +3,28 @@ package com.campudus.tableaux.database.domain
 import com.campudus.tableaux.database.model.TableauxModel._
 import org.vertx.scala.core.json._
 
-case class Table(id: TableId, name: String, hidden: Boolean) extends DomainObject {
-  override def getJson: JsonObject = Json.obj("id" -> id, "name" -> name, "hidden" -> hidden)
+case class Table(id: TableId, name: String, hidden: Boolean, langtags: Option[Seq[String]]) extends DomainObject {
+  override def getJson: JsonObject = {
+    val result = Json.obj(
+      "id" -> id,
+      "name" -> name,
+      "hidden" -> hidden
+    )
+
+    if (langtags.isDefined) {
+      result.mergeIn(Json.obj("langtags" -> langtags.orNull))
+    }
+
+    result
+  }
 }
 
 case class TableSeq(tables: Seq[Table]) extends DomainObject {
   override def getJson: JsonObject = Json.obj("tables" -> compatibilityGet(tables))
 }
 
 case class CompleteTable(table: Table, columns: Seq[ColumnType[_]], rowList: RowSeq) extends DomainObject {
-  override def getJson: JsonObject = table.getJson.mergeIn(Json.obj("columns" -> columns.map {_.getJson})).mergeIn(rowList.getJson)
+  override def getJson: JsonObject = table.getJson.mergeIn(Json.obj("columns" -> columns.map {
```

Comment:
Strange formatting. Would something like this look better / be better understandable?

```
override def getJson: JsonObject =
  table.getJson
    .mergeIn(Json.obj("columns" -> columns.map(_.getJson)))
    .mergeIn(rowList.getJson)
```